src/layout/Navbar/Navbar.scss

```
@@ -7,7 +7,8 @@
 }
 
 .navbar {
-	position: relative;
+	position: sticky;
+	top: 0;
```

use the logical property:
```suggestion
	inset-block-start: 0;
```